kbase-extension/static/kbase/js/common/cellComponents/tabs/results/outputWidget.js

```diff
@@ -119,23 +79,78 @@ define([
                     hidden: false,
                     type: 'default',
                     classes: ['kb-panel-container'],
-                    body: renderedOutput.layout,
+                    body: renderOutput(arg.objectData),
                 })
             );
+
             if (arg.objectData.length) {
                 const $objTable = $(ui.getElement('objects-table'));
+
+                const tableData = arg.objectData.map((obj) => {
+                    let name = obj.name;
+                    if (!name) {
+                        name = obj.wsInfo ? obj.wsInfo[1] : 'Unknown object name';
+                    }
+                    let type = obj.type;
+                    if (!type) {
+                        type = obj.wsInfo ? obj.wsInfo[2] : 'Missing type';
+                    }
+                    const parsedType = APIUtil.parseWorkspaceType(type) || { type };
+                    const description = obj.description || 'Missing description';
+                    return {
+                        wsInfo: obj.wsInfo,
+                        name,
+                        type,
+                        parsedType,
+                        description,
+                    };
+                });
+
                 $objTable.DataTable({
-                    searching: false,
-                    pageLength: tablePageLength,
+                    data: tableData,
+                    dom: "<'row'<'col-sm-12'tr>><'row'<'col-sm-5'i><'col-sm-7'p>>",
                     lengthChange: false,
-                    fnDrawCallback: () => {
-                        // Hide pagination controls if length is less than or equal to table length
-                        if (arg.objectData.length <= tablePageLength) {
-                            $(container).find('.dataTables_paginate').hide();
+                    pageLength: tablePageLength,
+                    paging: arg.objectData.length > tablePageLength,
```

Comment:
turn paging on or off, depending on whether it is required ==> don't need the `fnDrawCallback`